### PR TITLE
Use bullseye

### PR DESCRIPTION
**Pull request by bathina2**
## Change Overview

This PR switches to using debian bullseye (stable) vs buster and other alpine invocations.
I updated the build image I pushed last week. So no need to update the Makefile to point to a new version.

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues 

- fixes #issue-number

## Test Plan

- [ ] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

File: docker/kanister-elasticsearch/image/Dockerfile [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-FROM alpine:3.11.3
+FROM debian:bullseye
 COPY --from=TOOLS_IMAGE /usr/local/bin/restic /usr/local/bin/restic
 COPY --from=TOOLS_IMAGE /usr/local/bin/kopia /usr/local/bin/kopia
 ADD kando /usr/local/bin/
 
-ADD docker/kanister-elasticsearch/image/esdump-setup.sh /esdump-setup.sh
-RUN chmod +x /esdump-setup.sh && sync && /esdump-setup.sh
+RUN apt update
+RUN apt install -y nodejs npm bash curl
```
Comment (bathina2) [inline docker/kanister-elasticsearch/image/Dockerfile:10]: Was able to delete the shell script and directly call the install in the Dockerfile. Not sure why we used the shell script to begin with.

File: docker/postgres-tools-9.6/Dockerfile [hunk 2]
```diff
@@ -1,15 +1,15 @@
-FROM postgres:9.6-alpine
+FROM postgres:bullseye
```
Comment (pavannd1) [inline docker/postgres-tools-9.6/Dockerfile:1]: Ha! I was supposed to delete this image 8 months ago, right around the time I switched laptops I think 🤔 Might have a branch lying somewhere - will follow up 👍🏼

**Comment by julio-lopez** [comment]
+1: LGTM, deferring to @pavannd1 to make the call here.

File: pkg/testing/integration_register.go [hunk 3]
```diff
@@ -40,20 +40,21 @@ var _ = Suite(&PITRPostgreSQL{
...
-var _ = Suite(&PostgreSQL{
-	IntegrationSuite{
-		name:      "postgres",
-		namespace: "postgres-test",
-		app:       app.NewPostgresDB("postgres", ""),
-		bp:        app.NewBlueprint("postgres", "", true),
-		profile:   newSecretProfile(),
-	},
-})
+// TODO: Add back after https://github.com/kanisterio/kanister/pull/1705 merges
```
Comment (pavannd1) [inline pkg/testing/integration_register.go:43]: There's an easier way to disable tests :)
You could just remove them from this line - https://github.com/kanisterio/kanister/blob/master/build/integration-test.sh#L28

**Review by pavannd1 — APPROVED** [review]

File: pkg/testing/integration_register.go (hunk 3 above)
Comment (bathina2) [reply pkg/testing/integration_register.go:43]: Ah didn't see the shell script. Just followed the go stack up to the register

**Pull request merged**